test/e2e/tests/test_sagemaker_endpoint_autoscaling.py

```diff
@@ -227,8 +249,7 @@ def test_update(self, applicationautoscaling_client, generate_sagemaker_policy):
     def test_delete(self, applicationautoscaling_client, generate_sagemaker_policy):
         (
             resource_id,
-            target_spec,
-            target_reference,
+            target_reference, target_spec, target_resource,
```

Comment:
nit: strange formatting

Reply:
wasn't really ready for a review, was it :D

Reply:
Ran black.